config/root/clusterworkspacetype-team.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: tenancy.kcp.dev/v1alpha1
+kind: ClusterWorkspaceType
+metadata:
+  name: team
+spec:
+  initializers:
+  - tenancy.kcp.dev/team
+  defaultChildWorkspaceType: Universal
+  allowedChildWorkspaceTypes:
+    - Universal
```

Comment:
would not even have this here.

Reply:
Listing nothing allows no children. We must have it.

Reply:
I meant to put `"*"` here.